cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -1806,6 +1806,195 @@ search = | savedsearch "Palo Alto Firewall - Network Compromise - DDoS Attack Pr
+cron_schedule = 9,39 * * * *
+description = A Fortigate Firewall alerts for high or critical severity threats. \
+\
+Data Collection - Fortigate Networks firewalls traffic data (Fortinet Fortigate Add-on for Splunk).
+dispatch.earliest_time = -32m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_fortigate` sourcetype="fortigate_utm" severity IN ("high" "critical") \
+| eval cyences_severity = case( \
+    (action=="blocked" or action=="dropped") and (severity=="critical" or severity=="high" or severity=="medium" or severity=="low"), "low", \
+    (action=="blocked" or action=="dropped") and severity=="informational", "info", \
+    action=="allowed" and severity=="informational", "info", \
+    action=="allowed", severity) \
+| table _time, subtype, app, signature, severity, cyences_severity, action, dvc, src, src_port, srccountry, src_zone, dest, dest_port, url \
+| `cs_human_readable_time_format(_time, event_time)` \
+| `cs_fortigate_high_threats_alert_filter`
+action.cyences_notable_event_action = 1
+action.cyences_notable_event_action.param.filter_macro_name = cs_fortigate_high_threats_alert_filter
+action.cyences_notable_event_action.contributing_events = `cs_fortigate` sourcetype="fortigate_utm" severity IN ("high" "critical")
+action.cyences_notable_event_action.system_compromised_search = | stats sum(count) as count by dvc
+action.cyences_notable_event_action.system_compromised_drilldown = `cs_fortigate` sourcetype="fortigate_utm" severity IN ("high" "critical") dvc=$row.dvc$
+action.cyences_notable_event_action.attacker_search = | stats sum(count) as count, values(srccountry) as src_location by src
+action.cyences_notable_event_action.attacker_drilldown = `cs_fortigate` sourcetype="fortigate_utm" src=$row.src$
+action.cyences_send_email_action = 1
+action.cyences_notable_event_action.products = Fortigate
+
+[Fortigate Firewall - Network Compromise - Fortigate High System Alert]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 19,49 * * * *
+description = A Fortigate Firewall alerts for high or critical severity system events. Anything license-related is excluded from this alert as generally it's not that high severity. You can view them on the Fortigate dashboard.\
+\
+Data Collection - Fortigate firewalls system/events data (Fortinet Fortigate Add-on for Splunk).
+dispatch.earliest_time = -32m@m
+dispatch.latest_time = -2m@m
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_fortigate` sourcetype="fortigate_event" severity IN ("critical", "high") NOT "*license*" \
```

Comment:
Are we sure about this search?

Reply:
"fortigate_event" sourcetype is for system events (found from its data model which is categorize in datasets)

Not sure about license info. currently, no any event available related to license.